tools/scripts-core/utils/path-resolver.js

```diff
@@ -53,12 +56,25 @@ function getAbsolutePath(userPath) {
  * @param filePath The path
  * @returns {string} The relative path from cwd
  */
-function hereRelative(dirname, filePath) {
+export function hereRelative(dirname, filePath) {
 	return path.join(dirname, filePath).replace(process.cwd(), '.');
 }
 
-module.exports = {
-	getAbsolutePath,
-	hereRelative,
-	resolveBin,
-};
+export function getPkgRootPath(name) {
+	let rootPath;
+	try {
+		const indexPath = require.resolve(name);
+		let currentPath = indexPath;
+		let found = false;
+		while (!found) {
```

Comment:
Maybe it's because it's Monday but I don't see how the iteration work.
With my current understanding, there is a risk of infinite loop.

Reply:
path.dirname will return each time the parent folder